VirgilCryptoRatchet.podspec

```diff
@@ -13,8 +13,8 @@ Pod::Spec.new do |s|
   s.watchos.deployment_target   = "2.0"
   s.public_header_files         = "VirgilCryptoRatchet/VirgilCryptoRatchet.h"
   s.source_files                = "VirgilCryptoRatchet/**/*.{h,mm,swift}"
-  s.dependency 'VirgilCryptoFoundation', '= 0.15.2'
-  s.dependency 'VSCCrypto/Common', '= 0.15.2'
-  s.dependency 'VSCCrypto/Foundation', '= 0.15.2'
-  s.dependency 'VSCCrypto/Ratchet', '= 0.15.2'
+  s.dependency 'VirgilCryptoFoundation', '= 0.16.0-rc1'
```

Comment:
For production release it should be change to '~> 0.16.0'

Reply:
Fixed